frontend/src/App.vue

```diff
@@ -91,6 +97,9 @@ export default {
       _.set(prompt, "answers", answers)
       
       return prompt
+    },
+    uiMessages() {
+      return this.messages
```

Comment:
why there is a need for computed func here? we can use 'messages' object

Reply:
Fixed